src/DependencyInjection/DoctrineFixturesExtension.php

```diff
@@ -19,14 +21,33 @@ class DoctrineFixturesExtension extends Extension
      * {@inheritDoc}
      *
      * @return void
+     *
+     * @throws Exception
      */
-    public function load(array $configs, ContainerBuilder $container)
+    public function load(array $configs, ContainerBuilder $container): void
```

Comment:
This is a breaking change. We will include native return types in the 4.0.x release, but we cannot do this in a minor release.